src/scss/components/_form.scss

```diff
@@ -7,6 +7,34 @@
 **/
 
 .form {
+  position: relative;
+  overflow: hidden;
+
+  &__envelop {
+    background: get-color(reward);
+    width: 100%;
+    position: relative;
+    padding: 30px 30px 20px 30px;
+
+    &__title {
+      margin: 0;
+      text-align: center;
+      color: get-color(air);
+      text-transform: uppercase;
+      font-size: 14px;
+      font-weight: 900;
+    }
+  }
+
+  &__tie {
+    width: 100%;
+    height: 20px;
+
+    polygon {
+      fill: get-color(reward);
+      stroke: get-color(reward);
+    }
+  }
```

Comment:
@MarcoBrunoBR We already have a variable for this new borde-radius created on the button update. I think is better not to duplicate these changes on all the PR's. After the merge we can code review it again 👍